advanced/kernel-output/src/index.ts

```diff
@@ -30,27 +30,28 @@ namespace CommandIDs {
 const extension: JupyterFrontEndPlugin<void> = {
   id: 'kernel-output',
   autoStart: true,
-  requires: [ICommandPalette, ILauncher, IMainMenu, IRenderMimeRegistry],
+  optional: [ILauncher],
+  requires: [ICommandPalette, IMainMenu, IRenderMimeRegistry],
   activate: activate
 };
 
 function activate(
   app: JupyterFrontEnd,
   palette: ICommandPalette,
-  launcher: ILauncher,
   mainMenu: IMainMenu,
-  rendermime: IRenderMimeRegistry
+  rendermime: IRenderMimeRegistry,
+  launcher: ILauncher | null
 ) {
   const manager = app.serviceManager;
   const { commands, shell } = app;
-  let category = 'Tutorial';
+  const category = 'Example';
 
-  let panel: TutorialPanel;
+  let panel: ExamplePanel;
 
   function createPanel() {
     return manager.ready
       .then(() => {
-        panel = new TutorialPanel(manager, rendermime);
+        panel = new ExamplePanel(manager, rendermime);
```

Comment:
Should we use the `async` / `await` syntax to define the `createPanel` function?

Reply:
Agreed it helps readability